### PR TITLE
[risk=low][no ticket] rm UI logic based on FFs useGpu and usePersistentDisk

### PR DESCRIPTION
**Pull request by jmthibault79**
useGpu and usePersistentDisk are always true, and we are removing these Feature Flags (see #7592) so we can remove the UI logic which checks if they are false.

After a release, we can fully remove these FFs.

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

**Comment by jmthibault79** [comment]
The failing tests are not making a lot of sense.  Switching to Draft until we merge #7592 and can isolate this

**Comment by jmthibault79** [comment]
e2e tests have 4 failures, all common flakes.  Rerunning.

**Comment by jmthibault79** [comment]
e2e test run #2 - `controlled-tier/ct-workspace-share.spec.ts` failed.  Also a common flake, and none of the previous 4 failures.

File: ui/src/app/components/runtime-configuration-panel.tsx [hunk 1]
```diff
@@ -701,36 +699,17 @@ const PanelMain = fp.flow(
                       validMachineTypes={validMainMachineTypes}
                       machineType={analysisConfig.machine.name}
                     />
-                    {enablePersistentDisk || (
```
Comment (jmthibault79) [inline ui/src/app/components/runtime-configuration-panel.tsx:704]: This construction is a little odd.  I *think* what it's doing is: render this element when enablePersistentDisk is false (which is never)

**Review by evrii — APPROVED** [review]
Looks good to me.

**Pull request merged**